lib/AvirWrapper.php

```diff
@@ -179,6 +175,11 @@ function () use ($path, $logger) {
 						}
 
 						$owner = $this->getOwner($path);
+
+						// this will remove the file from the ui and respond to the sync client that the sync failed
+						$cache = $this->getCache($path);
+						$cache->remove($path);
+
 						$this->unlink($path);
```

Comment:
Didn't we check during debugging that we do not reach this point at all via the client upload? I can't reproduce that it works like this either. Am I missing something?

Reply:
yes. But I did debug again and reached that point. I guess, where was still something itchy on your configuration. Let's check that together today.